server/routes/creatingLicences/handlers/confirmation.ts

```diff
@@ -1,7 +1,28 @@
 import { Request, Response } from 'express'
+import LicenceType from '../../../enumeration/licenceType'
 
 export default class ConfirmationRoutes {
   GET = async (req: Request, res: Response): Promise<void> => {
-    res.render('pages/create/confirmation')
+    const { licence } = res.locals
+
+    let titleText
+    let confirmationMessage
+
+    switch (licence.typeCode) {
+      case LicenceType.AP_PSS:
+        titleText = `Licence and post sentence supervision order for ${licence.forename} ${licence.surname} sent`
+        confirmationMessage = `We have sent the licence and post sentence supervision order to ${licence.prisonDescription} for approval.`
+        break
+      case LicenceType.AP:
+        titleText = `Licence conditions for ${licence.forename} ${licence.surname} sent`
+        confirmationMessage = `We have sent the licence to ${licence.prisonDescription} for approval.`
+        break
+      case LicenceType.PSS:
+        titleText = `Post sentence supervision order for ${licence.forename} ${licence.surname} sent`
+        confirmationMessage = `We have sent the post sentence supervision order to ${licence.prisonDescription} for approval.`
+        break
+    }
```

Comment:
We're always sure its one of these from the enum, so ok without default.

Reply:
Could disable lint for default on this switch, rather then service wide? Ok for now though.